### ISSUE TITLE
weekly-timesheet-sharepoint - 

### ISSUE THREAD

**Issue by Scotty1629** [pnp/powerplatform-samples#95]
### Sample

weekly-timesheet-sharepoint

### Author(s)

@aprildunnam

### Question

hello, im new here. I've followed the guide to build the app but i can not submit detail when i test? i have now error? it just does nothing when i press submit or save?

**Comment by Scotty1629** [comment]
[Recording 2024-06-07 100631.zip](https://github.com/user-attachments/files/15728840/Recording.2024-06-07.100631.zip)

Correction I've found loads of errors ?

**Comment by sudhirke** [comment]
@Scotty1629 I encountered similar error and able to resolve it.  This was mostly due to field names in the SharePoint list.  "WeekStart" field in the list should be named with space "Week Start".  

So you have two fixes,  either keep the name as "WeekStart" and update all references in the code.  

I was able to get the solution to working,  Kept my field names without space and updated filed reference. Below is screen shot of my list 

<img width="528" alt="image" src="https://github.com/pnp/powerplatform-samples/assets/318482/87a4d520-472f-41b3-8059-9589dcf7db6d">
